### ISSUE TITLE
ox_inventory petrolcan and fireextinguisher bug

### ISSUE THREAD

**Issue by cornelyum** [overextended/ox_inventory#1446]
**Describe the bug**
- When I try to use the petrolcan or fireextinguisher they do not drain the ammo properly, I have 100 ammo representing a total of 2 ammo meanwhile the weapons having 4500 total ammo representing 100 ammo. However, when i use the 100 ammo it shows me it has, the weapon gets depleated and returned to the inventory but the ammo count in inventory is depleated by the small ammount consumed.

**Framework**
The framework your server uses for players (e.g. Ox, ESX, QB).
- ESX

**Resource version**
The version number listed in fxmanifest.lua, or optionally a commit hash.
- Version '2.32.3'

**To Reproduce**
1. Give yourself a petrolcan using '/giveitem 1 weapon_petrolcan 1'.
2. Equip the petrolcan in quickslot or straight up use it from the inventory.
3. Hold left click to pour gasoline on the ground.
4. The petrol in hand gets depleated in 1 second due to the ammo count it shows on top right corner being 2 or 100.
5. Look at the petrolcan in the inventory, it has depleated a small amount of ammo but still has more than enough left in it.
6. repeat the steps 2 and 3 and notice the same result.

**Expected behavior**
- I expected that the petrolcan and fireextinguisher when equiped would have the proper amount of ammo in it. By default they have 4500 ammo meaning a total of 100 ammo but when used the petrolcan while ox_inventory is installed it seems to instead limit the max ammo to 100 meaning a total of 2 ammo to use.

**Screenshots**
![image](https://github.com/overextended/ox_inventory/assets/142153193/e4f5caa9-ba48-43b3-8aa5-5d6e716c29cd)
![image2](https://github.com/overextended/ox_inventory/assets/142153193/8f9c9f3b-870d-4ce6-8e70-ad707bce11cf)

![image1](https://github.com/overextended/ox_inventory/assets/142153193/83d72b46-c9b5-446d-b908-27024ed962f7)

**Additional context**
- I tried multiple ways to fix the issue and got nowhere, it seems that the durability system for items like petrolcan or fireextinguisher is breaking them. Is there any way to remove durability completely? From all items?

**Comment by thelindat** [comment]
All weapons in `GROUP_PETROLCAN` and `GROUP_FIREEXTINGUISHER` use [SetPedInfiniteAmmo](https://docs.fivem.net/natives/?_0x3EDCB0505123623B), so I'm not sure how you would be running out.

**Comment by cornelyum** [comment]
I have no ideea what is causing it, I am still looking into it, are there any incompatibilities with the default esx_core scripts?

**Comment by thelindat** [comment]
No. Typically the conflicts with weapons are self-proclaimed "anticheats".

**Comment by DarkMinee** [comment]
Hi, I have the same issue, player can equip the petrolcan but after 1 second it starts aiming like a normal pistol with the petrolcan equipped. If I "fold" the petrolcan and I equip back again, same issue. I also stopped almost all resources to test out but nothing changed. I tried to search on the code and I noticied that in client.lua on line 1420 IsPedShooting(playerPed) return true for about 1 second and the false. I have no idea if it will help or not.
